tox.ini

```diff
@@ -40,7 +40,9 @@ setenv =
 commands =
     npm config set spin false
     npm install
-    node ./node_modules/gulp/bin/gulp.js build
+
+    npm run build
+    npm run lint
```

Comment:
nit: should lint first, so the travis run doesn't go through with the build before a possibly failing lint.